Form/Type/BasePickerType.php

```diff
@@ -87,20 +84,23 @@ protected function getCommonDefaults()
         return array(
             'widget'                   => 'single_text',
             'datepicker_use_button'    => true,
-            'dp_pick_time'             => true,
             'dp_use_current'           => true,
             'dp_min_date'              => '1/1/1900',
-            'dp_max_date'              => null,
-            'dp_show_today'            => true,
```

Comment:
we cannot remove the value, we need to be BC compatible. Just convert the values.